modules/config-posture/README.md

```diff
@@ -0,0 +1,67 @@
+# Oracle Cloud Config Posture Module
+
+This module will deploy Config Posture resources in Oracle for a compartment or root tenancy.
+
+The following resources will be created in each instrumented compartment/tenancy:
+
+- An Admit Policy on the target tenant that will allow sysdig tenant to `read` all-resources in the specified
+  compartment/tenancy.
+- A cloud account component in the Sysdig Backend, associated with the specified compartment/tenant and with the
+  required metadata to serve the Config Posture functions.
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+
+## Requirements
+
+| Name                                                                      | Version   |
+|---------------------------------------------------------------------------|-----------|
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1.0.0  |
+| <a name="requirement_oci"></a> [oci](#requirement\_oci)                   | >= 6.19.0 |
+| <a name="requirement_sysdig"></a> [sysdig](#requirement\_sysdig)          | >= 1.34.0 |
```

Comment:
nit: so that it doesn't get missed, this will be updated to allow minor version range e.g.`"~> 1.40"` once provider changes are released.

Reply:
@jose-pablo-camacho I believe this can be fixed in all places in this PR, and updated to `"~> 1.40"`, since the latest provider changes will be a minor version update and not major?

Reply:
I was waiting since we will need to have `1.41` instead, due to the policy_id change

Reply:
done